scrape.py

```diff
@@ -2,26 +2,36 @@
 from bs4 import BeautifulSoup
 
 
+def url_builder():
+    username = input("Enter user name: ")
+    url = 'https://www.codewars.com/users/' + username
+    return url
+
+
 def fetch(url):
     r = requests.get(url)
     soup = BeautifulSoup(r.content, 'html.parser')
-    user_info = soup.find_all('div', attrs={'class' : 'stat'})
+    user_info = soup.find_all('div', attrs={'class': 'stat'})
     counter = 0
+    
+    if not user_info:
+        print("Username does not exist, please try again. ")
+        return
```

Comment:
@Swaraj1998 if you are returning `None`, then I don't think you should say `please try again` because it simply exits from the program and there's no "trying again". You should probably change it to `Username doesn't exist, exiting...... `.

@Swaraj1998 @palash25 would like to know your opinions.

Reply:
@sibasish14 Well, "try again" could mean  "run the script again". But, anyway, either way works. I'll change it.